source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
+    <td nowrap><code>created</code></td>
+    <td>Your service made an API call to create the mandate.</td>
     </tr>
-    <td nowrap><code>STARTED</code></td>
-    <td>Your service sent an API request to GOV.UK Pay.</td>
+    <td><code>started</code></td>
+    <td>The user has visited the <code>next_url</code> and landed on the GOV.UK Pay
+page to enter their account information.</td>
     </tr>
-    <td nowrap><code>PENDING</code></td>
+    <td><code>pending</code></td>
     <td>Indicates that account details have been submitted to the
 payment service provider (PSP), but the PSP is not yet able to confirm whether
-the payment is complete.</td>
+the mandate has been successfully set up.</td>
+    </tr>
+    <tr>
+    <td><code>active</code></td>
+    <td>The PSP has confirmed that the mandate has been succesfully set up.</td>
+    </tr>    
+    <tr>
+    <td><code>inactive</code></td>
+    <td>You cannot collect payments from the mandate, for example because the user cancelled it.
+    </tr>    
+    <tr>
+    <td><code>cancelled</code></td>
+    <td>When the user entered their bank account information, they answered <b>No</b> to <b>Are you allowed to authorise Direct Debits on this account?</b>.</td>
+    </tr>   
   </table>
 </div>
 
-The payment reaches one of several finished states.
+## Payment statuses
 
 <br>
 <div style = "overflow-x:auto;">
   <table style = "width:100%">
     <tr>
-    <th nowrap>Finished state</th>
+    <th nowrap>State</th>
     <th>Description</th>
+    </tr>
+    <td nowrap><code>pending</code></td>
+    <td>The payment has been requested, but the PSP is not yet able to confirm whether
+the payment is complete.</td>
     </tr>
     <tr>
-    <td><code>SUCCESS</code></td>
-    <td>Payment succeeded.</td>
+    <td><code>success</code></td>
+    <td>The payment succeeded.</td>
     </tr>
-    <td><code>FAILED</code></td>
-    <td>Payment failed.</td>
+    <td><code>failed</code></td>
+    <td>The payment failed.</td>
     </tr>
-    <td><code>CANCELLED</code></td>
+    <tr>
+    <td><code>cancelled</code></td>
     <td>The service cancelled the payment.</td>
     </tr>
-    <td><code>ERROR</code></td>
-    <td>Something went wrong with GOV.UK Pay or the underlying Payment Service Provider.</td>
-    </tr>
   </table>
 </div>
 
-### Differences compared to card payment statuses
-
-The states `created`, `started` and `submitted` are all represented as
-a single state: `STARTED`.
-
-Direct Debit payments can have the non-terminal state `PENDING`.
-
 ## Reference numbers
 
 You can control what appears on paying users’ bank account statements.  You'll
-enter this when you set up your account with the Direct Debit supplier. This
-applies to both one-off and variable Direct Debit payments.
+enter this when you set up your account with GoCardless.
 
-The  Direct Debit supplier will generate a mandate reference number, which is
+GoCardless will generate a mandate reference number, which is
```

Comment:
It might be worth mentioning here that it’s a Direct Debit scheme requirement that this reference be given to user and that it may appear on their bank statement.

Reply:
I'll probably leave this out for now if that's ok - it's probably best to avoid getting too far into the service team's responsibilities around the Direct Debit scheme in the docs.